### PR TITLE
Service hours crud

### PR DESCRIPTION
**Pull request by qjd2413**

### CODE REVIEW

File: app/routers/service.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            res.sendStatus(400); 
+            return;
+          }
+
+          var hour = req.body;
+          //missing data
+          var isHours = hour.startTime && hour.endTime;
+          var isDonation = hour.amount;
+          var isBothOrNeither = isHours === isDonation;
+          if(!hour.description || isBothOrNeither) {
```
Comment (kalebingerick) [inline app/routers/service.js]: consolidate this line with the above

File: app/controllers/ServiceHourController.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  module.exports = {
+    findAll: function() {
+      return ServiceHour.findAll({
+        include: [Brother]
+      });
+    },
+    submit: function(brotherId, hours) {
+      hours.BrotherId = brotherId;
+      hours.state = 'u';
```
Comment (kalebingerick) [inline app/controllers/ServiceHourController.js]: Pull this out into two booleans: `looked_at` and `approved`. This way we aren't reliant on these characters

File: public/app/approveService/approveService.ctrl.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+
+(function() {
+
+  var approveServiceCtrl = function($scope, serviceService, approvableHours) {
+    $scope.done = approvableHours.length === 0;
+    $scope.currentHour = approvableHours.shift();
```
Comment (kalebingerick) [inline public/app/approveService/approveService.ctrl.js]: please make this more descriptive. `currentServiceHour` or something

**Comment by qjd2413** [comment]
@kalebdavis @Sean-Q-Sun @dustinsmartin can i merge this

File: public/app/submitService/submitService.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    End Time: <input type="text" ng-model="hour.endTime" /><br />
+  </div>
+
+  <div ng-show="type === 'Donation'">
+    Amount: <input type="text" ng-model="hour.amount" />
+  </div>
+
+  <button ng-click="submit()">Submit</button>
+
+  <div ng-show="success">Hour successfully submitted.</div>
```
Comment (kalebingerick) [inline public/app/submitService/submitService.html:22]: Should say 'Service successfully submitted', because it's not always an hour

**Pull request merged**